git_buildpackage

```diff
@@ -163,9 +172,27 @@ def create_source_package(repo_dir, output_dir, revision, build_args):
     safe_run(command, cwd=repo_dir, interactive=True)
 
 
+    # Lets find the .dsc
+    #if not os.path.exists(
+    found_dsc = False
+    for fname in os.listdir(absoutput_dir):
+        if fnmatch.fnmatch(fname, '*.dsc'):
+            input_file = os.path.join(absoutput_dir, fname)
+            found_dsc = True
+            with open(input_file, 'a') as dsc_file:
+                dsc_file.write('Git-Id: ' + git_id)
+                dsc_file.write('Git-URL: ' + url)
```

Comment:
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256

Hi Daniel,

I've modified the arguments, here is an example:

./git_buildpackage --url
ssh://git@swnstash.brocade.com:7999/~cdochert/brit.git --verbose
- --outdir ./tmp --gitid enable --giturl disable

I'll push an update once I've done some more testing. Let me know if
your happy. The default for both new arguments is "enable".

Thanks,
Colin.


On 04/21/2016 06:33 AM, Daniel Gollub wrote:
> In git_buildpackage 
> <https://github.com/gollub/obs-service-git_buildpackage/pull/1#discuss
ion_r60528145>:
>
> 
>> @@ -163,9 +172,27 @@ def create_source_package(repo_dir,
>> output_dir, revision, build_args): safe_run(command,
>> cwd=repo_dir, interactive=True)
>> 
>> 
>> +    # Lets find the .dsc +    #if not os.path.exists( +
>> found_dsc = False +    for fname in os.listdir(absoutput_dir): +
>> if fnmatch.fnmatch(fname, '*.dsc'): +            input_file =
>> os.path.join(absoutput_dir, fname) +            found_dsc = True 
>> +            with open(input_file, 'a') as dsc_file: +
>> dsc_file.write('Git-Id: ' + git_id) +
>> dsc_file.write('Git-URL: ' + url)
> 
> Could you make the exposing of the Git-URL configurable?
> 
> — You are receiving this because you authored the thread. Reply to
> this email directly or view it on GitHub 
> <https://github.com/gollub/obs-service-git_buildpackage/pull/1/files/d
caaaf13ae8364b587058c1b4e7ffc17e55de7bd#r60528145>
>
-----BEGIN PGP SIGNATURE-----
Version: GnuPG v2

iQIcBAEBCAAGBQJXGJhqAAoJEFFTL+lu4EfoVOIP/21tMXV4gSrHmWxYKZSlsA19
PLyLHGn6W6mw/6Kirroz6vE4DbNVqbQO+Dy4x83r018ZJXBjG6d8h/eI7q8FbHFg
48KWfwsmDDOdL3imYBxwTXQTC9mFtfITKcJNMfixKwlEV1+xSWG7u6M5eVBsz6Wz
qW9Bk2nP4TYU/NbmniYF2/Sbc4wQLT4v9IS88HGnVBJBI+5fLn9H2BmmZGBhollg
KJKtvfp4wjOqvHZ9u11eS59sAaKLLRneD/Wwpo9T0kRh1g1e8j2Ppt6lG+u5sSRu
wq++/N2/Y8ZsZ+zjIgphi3vZw2cwfBWv007PRNL1h4MlpJergh6uMzHN5VRVGgNb
Y1L9dKWTDX7M8zbLvH+9Bet1T8GSMIb45LZAeAvgInTRCqKk3PBCwgm3QgVN+bhN
kff9V81ECW0qbH5fRnLUb6vYaIr96PngkHXcOlKoggMrTMnai2tV1K6Dy2Gx5Ab0
N2biqn6MAnAxpVCA9oUwyulY+YHw9OGGtVSWkaQ3pEZUZdvxSOejuHJpnJOg+xxP
Pc0A53CywOF4Lbl6jV5AhcMi2DuAMx1+CbNGzrjst1C7ryA2rjdzoCoJZnfoNl8y
1/aGfwKiKdnjv8RmfVa92vu/BVOlCLkanApJMHt86/4Q9k35NEzdacfQFyCzxcqg
cO9L3wD4Ia9TWJwA7YPD
=hg8D
-----END PGP SIGNATURE-----